waku/v2/node/wakunode2_test.go

```diff
@@ -318,3 +335,94 @@ func TestDecoupledStoreFromRelay(t *testing.T) {
 	require.Len(t, result.Messages, 1)
 	require.Equal(t, msg.Timestamp, result.Messages[0].Timestamp)
 }
+
+func TestStaticShardingMultipleTopics(t *testing.T) {
+	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
+	defer cancel()
+
+	// Node1 with Relay
+	hostAddr1, err := net.ResolveTCPAddr("tcp", "0.0.0.0:0")
+	require.NoError(t, err)
+	wakuNode1, err := New(
+		WithHostAddress(hostAddr1),
+		WithWakuRelay(),
+	)
+	require.NoError(t, err)
+	err = wakuNode1.Start(ctx)
+	require.NoError(t, err)
+	defer wakuNode1.Stop()
+
+	pubSubTopic1 := protocol.NewStaticShardingPubsubTopic(uint16(0), uint16(0))
```

Comment:
ClusterID 0 cannot be used for static sharding. 0 would be considered as non-sharded default pubsubTopic.

Refer rfc https://rfc.vac.dev/spec/52/ for more details on shard allocation and usage.

Reply:
Are all 0-15 cluster index IDs reserved -> cannot be used with static sharding ?

Reply:
Reserved means can be used with static-sharding. It is just that someone else should not use these clusters other than Status/Waku.
But 0 is not supposed to be used for static-sharding as it is marked as global.